app/models/way_of_being.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+class WayOfBeing < ApplicationRecord
+  has_many :student_way_of_being, dependent: :destroy
```

Comment:
```suggestion
  has_many :student_ways_of_being, dependent: :destroy
```
has_many should be plural